.storybook/webpack.config.js

```diff
@@ -57,15 +57,19 @@ module.exports = ({ config }) => {
         options: {
           importLoaders: 1,
           modules: useCssModules,
-          localIdentName: '[local]-[hash:base64:5]',
+          // localIdentName: '[local]-[hash:base64:5]',
```

Comment:
One TODO: check if we still need the localIndentName. It works fine without it right now as can see it operates but would be good to check why it was added/needed